internal/ng_apf_library/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//internal:__subpackages__"])
+
+bzl_library(
+    name = "bzl",
+    srcs = glob(["*.bzl"]),
+    visibility = ["//visibility:public"],
+)
+
+filegroup(
+    name = "package_contents",
```

Comment:
seems to be unreferenced?

I don't think we need to document this rule, since it won't be "public api" (not re-exported through the index.bzl)
can delete most of this file?

Reply:
this is now included in `rules_nodejs_package` npm_package in the root BUILD file - I forgot to add it in the earlier commit.